dashboard/src/components/AppView/AccessURLTable/AccessURLTable.tsx

```diff
@@ -166,7 +182,13 @@ export default function AccessURLTable({ ingressRefs, serviceRefs }: IAccessURLT
       .concat(
         allIngresses.map(ingress => {
           return {
-            url: ingress.item ? getAnchors(GetURLItemFromIngress(ingress.item).URLs) : "Unknown",
+            url: ingress.item
+              ? GetURLItemFromIngress(ingress.item).URLs.map(
+                  // check whether each URL is, indeed, a valid URL.
+                  // If so, render the <a>, othersiwe, render a simple <span>
+                  url => (IsURL(url) ? getAnchor(url) : getSpan(url)),
```

Comment:
great, can you create a test in AccessURLTable.test for covering the case in which the `path` is a regexp? You can verify that there is no anchor in that case.

Tip: You can copy the test setup from "should show the table with available ingresses"